R/check_frontmatter.R

```diff
@@ -73,65 +74,82 @@ check_frontmatter <- function(
   }
   author_name <- map_lgl(yml_protocol$author, ~is.string(.$name))
   author_orcid <- map_lgl(yml_protocol$author, ~is.string(.$orcid))
-  problems <- c(problems,
-                sprintf(
-                  "Author name %s is not a string",
-                  map_chr(yml_protocol$author, "name")[!author_name]
-                ))
-  problems <- c(problems,
-                sprintf(
-                  "Author orcid %s is not a string",
-                  map_chr(yml_protocol$author, "orcid")[!author_orcid]
-                ))
+  problems <-
+    c(problems,
+      sprintf(
+        "Author nr %s had an invalid name (no string)",
+        seq_len(length(author_name))[!author_name]
```

Comment:
```suggestion
        seq_along(author_name)[!author_name]
```
The same but shorter.